Description: there needs to be a UI for changing the current user's password,
 that can be launched externally.  Currently, this is the best work-around
 available.
Ubuntu: https://launchpad.net/bugs/307019

Index: gnome-control-center-2.29.4/capplets/about-me/gnome-about-me.c
===================================================================
--- gnome-control-center-2.29.4.orig/capplets/about-me/gnome-about-me.c	2010-01-11 02:16:27.000000000 +0000
+++ gnome-control-center-2.29.4/capplets/about-me/gnome-about-me.c	2010-01-11 02:18:34.000000000 +0000
@@ -1073,6 +1073,12 @@
 	rc = about_me_setup_dialog ();
 
 	if (rc != -1) {
+		if (argc > 1 && strcmp(argv[1],"--password")==0) {
+			GtkBuilder *dialog = me->dialog;
+			gtk_widget_hide_all ( WID ("about-me-dialog") );
+			about_me_passwd_clicked_cb (NULL, me);
+			return rc;
+		}
 		gtk_main ();
 	}
 
